rootfs/api/models/release.py

```diff
@@ -24,6 +24,7 @@ class Release(UuidAuditedModel):
     version = models.PositiveIntegerField()
     summary = models.TextField(blank=True, null=True)
     failed = models.BooleanField(default=False)
+    deployed = models.BooleanField(default=True)
```

Comment:
> any check to see if the release is deployed should be a call out to kubernetes...

That sounds more robust. We have run into issues trying to mirror cluster state in the database before (see the now-defunct `Containers` model for an example).